### ISSUE TITLE
Suggestion: Guide maker tool

### ISSUE THREAD

**Issue by malloot** [shagu/ShaguQuest#3]
Dear Eric Mauser

Thank you very much for your db and quest tool, i would be spending allot more time leveling without it!

I would like to create a leveling guide using your database/quest tool, however i am no programmer and creating a framework for a crazy arrow to use your DB is not really within the scope of my skills. However writing the guide using the framework would be very doable. 

Is there a chance you could create a guide making framework for your DB so people can easily create guides for events, quest chains and leveling? 
Basically like Vannilaguide but then with getting the locations of mobs, npc's and vendors from your db, and setting way points for a crazy arrow that way. 

Thank you in advance for any answer. 

Greetings 
David Kohnstamm

**Comment by shagu** [comment]
Creating a Guide Addon would be possible. But I don't know how hard it is to create an arrow which points to the correct directions. The Framework could look like this:
[Questname] = { [Item1] = "Required Item"; [Item2] = "Required Item2"; },
[nextQuestname] = { [spawn1] = "Required Kill"; [area1] = "42,71,10"; },
...
The addon could then listen on Quest Update events and automatically trigger ShaguDB via
/shagu [clean,spawn,item,quest] commands to display the proper things.

But I don't have the time to create such questing or profession guides.
Please send me an email, if you're intrested in doing such.

**Comment by Schaka** [comment]
Our backport of the CrazyTaxi Arrow isn't fully functionaly as a separate addon from Questie. However, the version in Questie 2.0 could be exported fairly easily. 

The main problem is passing it the correct zone, map and x,y values for it to point where you want it to.
